service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadTranslationsSubmittedControllerTest.java

```diff
@@ -0,0 +1,284 @@
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.Constants.LOCAL_AUTHORITY_1_CODE;
+import static uk.gov.hmcts.reform.fpl.Constants.LOCAL_AUTHORITY_1_INBOX;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.ITEM_TRANSLATED_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.AssertionHelper.checkUntil;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.OrderIssuedNotificationTestHelper.getExpectedParametersMap;
+import static uk.gov.hmcts.reform.fpl.utils.OrderIssuedNotificationTestHelper.getExpectedParametersMapForRepresentatives;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.documentSent;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.printRequest;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testAddress;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocument;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentBinaries;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
+import static uk.gov.hmcts.reform.fpl.utils.matchers.JsonMatcher.eqJson;
+
+@WebMvcTest(ManageOrdersController.class)
+@OverrideAutoConfiguration(enabled = true)
+class UploadTranslationsSubmittedControllerTest extends AbstractCallbackTest {
+
+    private static final Long CASE_ID = 1614860986487554L;
+    private static final String NOTIFICATION_REFERENCE = "localhost/" + CASE_ID;
+    private static final String FAMILY_MAN_CASE_NUMBER = "FMN1";
+    private static final UUID LETTER_1_ID = randomUUID();
+    private static final UUID LETTER_2_ID = randomUUID();
+    private static final Document ORDER_DOCUMENT = testDocument();
+    private static final Document COVERSHEET_REPRESENTATIVE = testDocument();
+    private static final Document COVERSHEET_RESPONDENT = testDocument();
+    private static final byte[] ORDER_BINARY = testDocumentBinaries();
+    private static final byte[] COVERSHEET_REPRESENTATIVE_BINARY = testDocumentBinaries();
+    private static final byte[] COVERSHEET_RESPONDENT_BINARY = testDocumentBinaries();
+    private static final DocumentReference ORDER = testDocumentReference();
+    private static final String ORDER_TYPE = "Care order (C32A)";
+    private static final Map<String, Object> NOTIFICATION_PARAMETERS = getExpectedParametersMap(ORDER_TYPE, true);
```

Comment:
I am confused about this, this is used for the order issuing event, it contains a json object as we occasionally attach a document through gov notify. Your email template doesn't have any of that so it should fail

Reply:
the matcher qqjson() was messing up. Moved to eq()